src/main/java/frc/robot/commands/drive/TeleopSwerve.java

```diff
@@ -47,9 +50,15 @@ public void execute() {
         xaxis = Conversions.applySwerveCurve(xaxis, Constants.STICK_DEADBAND);
         raxis = MathUtil.applyDeadband(raxis, Constants.STICK_DEADBAND);
 
-        Translation2d translation =
-            new Translation2d(yaxis, xaxis).times(Constants.Swerve.MAX_SPEED);
-        double rotation = raxis * Constants.Swerve.MAX_ANGULAR_VELOCITY;
+        double angle_speed = Constants.Swerve.MAX_ANGULAR_VELOCITY;
+        double speed = Constants.Swerve.MAX_SPEED;
+        if (arm.getArmAngle() > -20) {
+            angle_speed /= 2;
+            speed *= 0.60;
+        }
```

Comment:
I assume this worked well?

Reply:
We tested it in the pits. I assume it worked out well. It was running throughout playoffs.